hedera/views.py

```diff
@@ -54,3 +55,7 @@ def after_signup(self, form):
         profile = self.created_user.profile
         profile.display_name = form.cleaned_data["display_name"]
         profile.save()
+
+
+class DashboardView(TemplateView):
+    template_name = "dashboard.html"
```

Comment:
Since this is so simple, I support including the dashboard view in `hedera.views` at this point. But if if it starts to get more complicated, we could consider moving it to its own app.

Reply:
I wonder though if this should be a authenticated-only view. If someone hits `/dashboard/` unauth'd we probably want to redirect them to login rather than give a dashboard with silent errors because `request.user` is not a proper User.

Reply:
Yeah, the url path should be wrapped in `login_required`

Reply:
Makes sense! Will update the PR